src/common/components/share/styles.css

```diff
@@ -7,6 +7,7 @@
   background-color: #666;
   background-position: 50% 50%;
   background-repeat: no-repeat;
+  margin-right:6px;
```

Comment:
`CopyToClipboard` is reused in different context and this additional margin unnecessarily move it a bit.